homeassistant/components/notion/strings.json

```diff
@@ -10,11 +10,11 @@
       }
     },
     "error": {
-      "invalid_credentials": "Invalid username or password",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
       "no_devices": "No devices found in account"
```

Comment:
I did not changed this one with [%key:common::config_flow::error::no_devices_found%] because i don't find it relevant here. Please advise !

Reply:
How do you mean, relevant?